view/index.css

```diff
@@ -1,32 +1,22 @@
-table {
-    border-collapse: collapse;
-}
-
-table,
-th,
-td {
-    border: 1px solid black;
+.box {
+background-color: #009988;
+    margin: 15px;
 }
 
-.divTable {
-    display: table;
-    width: auto;
-    background-color: #eee;
-    border: 1px solid #666666;
-    border-spacing: 5px;
-    /*cellspacing:poor IE support for  this*/
-    /* border-collapse:separate;*/
-}
-
-.divRow {
-    display: table-row;
-    width: auto;
+.row {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
```

Comment:
No need to `wrap` anything here, since we want single row to be in single line only.